src/app/EmployeesHistory/Components/employeeDetails.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ProjectsHistoryDetails from './projectsHistoryDetails';
```

Comment:
Maybe having a convention file in the project will help us to follow best practice rules.